server/pxf-api/src/main/java/org/greenplum/pxf/api/model/RequestContext.java

```diff
@@ -63,25 +68,101 @@ public void setRequestType(RequestType requestType) {
     }
 
     // ----- NAMED PROPERTIES -----
+
+    /**
+     * The fully-qualified class name for the java class that was defined as
+     * Accessor.
+     */
     private String accessor;
```

Comment:
```suggestion
    private String accessorClassName;
```
should we do this and for other components as well (fragmented / resolver) ?

Reply:
I do not want to make any changes that will impact other classes. This will make us change getters for accessor, fragmenter, resolver.